apartments/tests.py

```diff
@@ -1,85 +1,47 @@
-from users.models import User
 from apartments.models import Apartment, City
 from .forms import ApartmentCreationForm
 from django.urls import reverse
 import pytest
 
 
-@pytest.fixture
-def new_user():
-    user = User.objects.create_user(
-        email='test@test.com',
-        first_name='first_name',
-        last_name='last_name',
-        birth_date='1900-01-01',
-        password='password'
-    )
-    user.save()
-    return user
-
-
-@pytest.fixture
-def new_city():
-    city = City(cityName='some_city')
-    city.save()
-    return city
-
-
-@pytest.fixture
-def new_apartment(new_user, new_city):
-    apartment = Apartment(
-        owner=new_user,
-        city=new_city,
-        address='street',
-        rent=4500,
-        num_of_roomates=2,
-        num_of_rooms=3,
-        start_date='2021-1-1',
-        about='Hey!',
-        image_url='www.some-url.com',
-    )
-    apartment.save()
-    return apartment
-
-
 @pytest.mark.django_db
 class TestModels:
 
-    def test_city_creation(self, new_city):
-        assert new_city is not None
-        assert new_city.cityName == 'some_city'
+    def test_city_creation(self, city_model):
+        assert city_model is not None
+        assert city_model.cityName == 'nice_city'
```

Comment:
this implicitly verifies that city_model is not None, so no need to explicitly check .
If city_model is None (or any object that doesn't have a city_name attribute) you'll get an AttributeError exception
`assert city_model is not None`
As an aside, if you're testing for not None you can leave that part out `assert city_model`

Reply:
@simzacks Those changes are part of the #147 PR which this PR is dependent on.
I didn't write any new tests just moved fixtures to conftest.py

nevertheless I'll look at the changes you requested and change the code accordingly in #147